tardis/apps/publication_workflow/components/PublicationButton.jsx

```diff
@@ -0,0 +1,31 @@
+import PropTypes from 'prop-types';
+import React, { Fragment } from 'react';
+
+const PublicationButton = ({ onclick, setViewType }) => (
+  <Fragment>
+    <div className="row">
+      <div className="col-md-12 pull-right">
+        <button type="button" className="btn btn-primary btn-lg mb-2 ml-2 pull-right" onClick={onclick}>
+          <i className="fa fa-plus mr-2" />
+          Create publication
+        </button>
+        <div className="pull-right btn-group btn-lg">
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 16 spaces but found 8.](https://app.codacy.com/gh/mytardis/mytardis/pullRequest?prid=7933947)
```suggestion
                <div className="pull-right btn-group btn-lg">
```